slides/mentoring.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+
+    <title>Mentoring abroad</title>
+
+    <link rel="stylesheet" href="../css/reveal.css" />
+    <link rel="stylesheet" href="../css/theme/fac.css" />
+
+    <!-- Theme used for syntax highlighting of code -->
+    <link rel="stylesheet" href="../lib/css/zenburn.css" />
+
+    <!-- Printing and PDF exports -->
+    <script>
+      var link = document.createElement("link");
+      link.rel = "stylesheet";
+      link.type = "text/css";
+      link.href = window.location.search.match(/print-pdf/gi)
+        ? "../css/print/pdf.css"
+        : "../css/print/paper.css";
+      document.getElementsByTagName("head")[0].appendChild(link);
+    </script>
+
+    <style>
+      /* BLACK: #121212 */
+      /* YELLOW: #ffe500 */
+      /* BLUE: #0805ec */
+      /* GREEN: #00be00 */
+      /* RED: #ff2800 */
+    </style>
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section data-background-color="#121212">
+          <h1>Mentoring abroad ✨</h1>
+        </section>
+
+        <section>
+          <section>
+            <h1>
+              <span class="bigunderline bigunderline--blue">Where are our campuses?</span>
+            </h1>
+            <p class="fragment">
+              We actually have just one campus, here in London 💂
+            </p>
+            <aside class="notes">
+            </aside>
+          </section>
+
+          <section data-background-image="../assets/photos/FAC15.jpg">
+            <h2>
+              <span style="background: rgba(255,255,255,0.5)">London</span>
+            </h2>
+            <p class="fragment">
+                <span style="background: rgba(255,255,255,0.5)">We started around 2012 in Camden and we're now on our 17th (you!) cohort.</span> 
```

Comment:
Replace "we're now on our 17th (you!) cohort" with "we've come a long way", perhaps. So, we don't have to update this bit every 4 months.

Reply:
👍